bin/release

```diff
@@ -1,56 +0,0 @@
-#!/usr/bin/env bash
-
-# TODO: prepare changelog
```

Comment:
Why did you remove the file?

Reply:
@makasim I just ran `git rm -rm . —cached && git add .`
It seems that the file matches one of the gitignore patterns
I can fix the gitignore though if it’s required.

Reply:
I've reverted the deletion of this file